internal/kms/kes.go

```diff
@@ -450,3 +472,34 @@ func (c *kesClient) Verify(ctx context.Context) []VerifyResult {
 	}
 	return results
 }
+
+// ValidateKey checks the validity of the KMS Master Key
+func (c *kesClient) ValidateKey(ctx context.Context) bool {
+	c.lock.RLock()
+	defer c.lock.RUnlock()
+
+	validKey := false
+	kmsContext := Context{"MinIO admin API": "ServerInfoHandler"} // Context for a test key operation
+	for _, endpoint := range c.client.Endpoints {
+		client := kes.Client{
+			Endpoints:  []string{endpoint},
+			HTTPClient: c.client.HTTPClient,
+		}
+
+		// 1. Generate a new key using the KMS.
+		kmsCtx, err := kmsContext.MarshalText()
+		if err != nil {
+			validKey = false
+			break
+		}
+		_, err = client.GenerateKey(ctx, env.Get(EnvKESKeyName, ""), kmsCtx)
+		if err != nil {
+			validKey = false
+			break
+		}
```

Comment:
I think we must use loggerOnce here to log use something like

```go
func kmsLogOnceIf(ctx context.Context, err error, id string, errKind ...interface{}) {
        logger.LogOnceIf(ctx, "kms", err, id, errKind...)
}
```

Add this function to `cmd/logging.go` and use it here

```
kmsLogOnceIf(ctx, err, "refresh-kms-master-key")
```

Reply:
Implemented